code/modules/reagents/chemistry/reagents/food.dm

```diff
@@ -590,13 +628,15 @@
 	description = "Bread! Yep, bread."
 	reagent_state = SOLID
 	color = "#9C5013"
+	taste_message = "bread"
```

Comment:
"Tastes like the best thing since sliced- wait."

Don't actually, I just wanted to try making a joke